rails/presenters-pattern/app/views/_navbar.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="main-content">
+  <ul class="nav nav-steps">
+    <%= k(:navbar).section_for(:personal_info) %>
```

Comment:
now that we are going to standardize this I'm not so sure about using `k` I would rather use `presents`

Reply:
Good point. I have always preferred `k` because it's short. But it's a very bad naming. `k` stands for keynote that is the name of the library used. 

I think that we can live with 8 more chars per line 😅